### PR TITLE
Burst shotgun begone

### PR DESCRIPTION
**Pull request by Kuro-ai**
## About The Pull Request

Groundside double barrel doesn't have burst fire anymore.

## Why It's Good For The Game

Burst shotguns are bad balance,one shot memes begone.

## Changelog
:cl:
balance:groundside double barrel doesn't have burst anymore
/:cl:

### CODE REVIEW

**Comment by ilikesaying** [comment]
muh supershotgun

**Comment by BraveMole** [comment]
thanks, retarded shotgun begone

File: code/modules/projectiles/guns/shotguns.dm [hunk 1]
```diff
@@ -296,7 +296,7 @@ can cause issues with ammo types getting mixed up during the burst.
 	attachable_offset = list("muzzle_x" = 33, "muzzle_y" = 21,"rail_x" = 15, "rail_y" = 22, "under_x" = 21, "under_y" = 16, "stock_x" = 21, "stock_y" = 16)
 
 	fire_delay = 2
-	burst_amount = 2
+	burst_amount = 1
```
Comment (SplinterGP) [inline code/modules/projectiles/guns/shotguns.dm:299]:
```suggestion
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by SplinterGP — APPROVED** [review]
it can still fire fast asfuck btw, change its firerate later to 0.4 seconds

**Pull request merged**